cdap-common/bin/functions.sh

```diff
@@ -415,16 +417,16 @@ cdap_set_hbase() {
     "") die "Unable to determine HBase version! Aborting." ;;
     *-cdh*)
       if [[ $(cdap_get_conf "hbase.version.resolution.strategy" "${CDAP_CONF}"/cdap-site.xml auto.strict) == 'auto.latest' ]]; then
-        __compat="${CDAP_HOME}"/hbase-compat-1.2-cdh5.7.0/lib/* # if allowed, return the latest CDH HBase version for unsupported HBase version
+        __compat="${CDAP_HOME}"/"${__latest_hbase_compat_cdh}"/lib/* # if allowed, return the latest CDH HBase version for unsupported HBase version
```

Comment:
There's no reason for quotes around your variable. It's local read-only content that isn't user-supplied, so it'll never have a space or anything.